neural_compressor/compression/pruner/pruners/pattern_lock.py

```diff
@@ -0,0 +1,59 @@
+"""pattern lock pruner."""
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base import (register_pruner,
+                   PytorchBasePruner,
+                   KerasBasePruner)
+from ..patterns import get_pattern
+
+
+@register_pruner('pt_pattern_lock')
```

Comment:
If pytorch pruners are prefixed with 'pt', all our existing example configs need to be updated.

Reply:
No. This situation has been considered. For all these existing example, there is no different between this version and previous one. API change only happen under compression/pruner and for outside coding, the api is the same. The get_pruner function is exactly the same with previous version. Using parameter framework to return pytorch or keras pruners.